idl/chimera.pro

```diff
@@ -0,0 +1,535 @@
+;+
+; Project     : Coronal Hole Identification via Multi-thermal Emission Reconition Algorithm (CHIMERA)
```

Comment:
No problem :)
Regarding breaking the file into multiple functions, you could have the functions inside the same file, and don't get all the problems on which file to run.